lib/processors/webgl2/helpers/webglHelper.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * Use it along with boyswan.glsl-literal VSCode extension
+ * to get GLSL syntax highlighting.
+ * https://marketplace.visualstudio.com/items?itemName=boyswan.glsl-literal
+ *
+ * On VSCode OSS, boyswan.glsl-literal requires slevesque.shader extension
+ * to be installed as well.
+ * https://marketplace.visualstudio.com/items?itemName=slevesque.shader
+ */
+export const glsl = String.raw
+
+export function createPiplelineStageProgram(
```

Comment:
Can we make sure the webgl files follow our programming conventions, like semicolons at the end of the line, etc.?

Reply:
It's one of my notes in the PR which I lowered in priority due to time constraints.